desktop/release-notes.md

```diff
@@ -24,35 +24,89 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/general.md/#releases)
 
+## 4.18.0
+
+{% include release-date.html date="2023-04-03" %}
+
+> Download Docker Desktop
+>
+> {% include desktop-install.html %}
+
+### New
+
+- Initial beta release of `docker init` as per [the roadmap](https://github.com/docker/roadmap/issues/453).
+- Added a new **Learning Center** tab to help users get started with Docker.
+
+### Upgrades
+
+- [Buildx v0.10.4](https://github.com/docker/buildx/releases/tag/v0.10.4)
+- [Compose 2.17.2](https://github.com/docker/compose/releases/tag/v2.17.2)
+- [Containerd v1.6.18](https://github.com/containerd/containerd/releases/tag/v1.6.18), which includes fixes for [CVE-2023-25153](https://github.com/advisories/GHSA-259w-8hf6-59c2) and [CVE-2023-25173](https://github.com/advisories/GHSA-hmfx-3pcx-653p).
+
+### Bug fixes and enhancements
+
+#### For all platforms
+
+- [Docker Scout CLI](../scout/index.md#docker-scout-cli) can now compare two images and display packages and vulnerabilities differences. This command is in [Early Access](../release-lifecycle.md) and might change in the future.
+- [Docker Scout CLI](../scout/index.md#docker-scout-cli) now displays base image update and remediation recommendations using `docker scout recommendations`. It also displays a short overview of an image using `docker scout quickview` commands.
+- You can now search for extensions direct from the Marketplace, as well as using **Global Search**.
+- Fixed a bug where `docker buildx` container builders would lose access to the network after 24hrs.
```

Comment:
Just realized that the Compose Watch line isn't in here yet!

Ideally we'd want to link the new docs on Watch, but the [PR](https://github.com/docker/docs/pull/17010) isn't yet merged.

```suggestion
- Docker Compose now has an experimental file watch command that will automatically update your running Compose services as you edit & save your code.
- Fixed a bug where `docker buildx` container builders would lose access to the network after 24hrs.
```

Reply:
Or should we move that into the `### New` section similar to the `docker init`?